src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -945,6 +954,10 @@ private TrackBitrateAllocation(
             int preferredIdx = 0;
             for (RTPEncodingDesc encoding : encodings)
             {
+                if (encoding.getHeight() > this.maxFrameHeight)
```

Comment:
Right now it works because of the way the height is discovered/set. But, in the future, where the height is either discovered from the RTP payload or from signaling from the client, it won't. I'm thinking it's more appropriate to change this into a resolution field, which would hold the multiplication result.